wikidict/lang/fr/__init__.py

```diff
@@ -738,6 +737,12 @@ def last_template_handler(
             return f"{author}, {italic(book)}, {date}"
         return f"{author}, {italic(book)}, {date}, page {page}"
 
+    if tpl == "R":
+        if parts[0] == "TLFi":
+            return f"«&nbsp;{word}&nbsp;», dans <i>TLFi, Le Trésor de la langue française informatisé</i>, 1971–1994"
+        else:
+            return ""
```

Comment:
```suggestion
        if not parts:
            return ""
        if parts[0] == "TLFi":
            return f"«&nbsp;{word}&nbsp;», dans <i>TLFi, Le Trésor de la langue française informatisé</i>, 1971–1994"
        return ""
```
Suggesting that code to a have clear view. We may need to add more conditions later.

Reply:
Not sure if sourcery will like that. Let's try